### DISCUSSION TITLE
Winter simulation

### DISCUSSION THREAD

**Discussion by SimoneBeltramino** [UMEP-dev/UMEP#413] [General]
I have a problem understanding the result of simulations in winter conditions. The result with the same input spatial data in summer conditions works while in winter conditions I get strange results. 
Attached is the photo that compares two points analyzed with SOLWEIG and ENVI-met with the same conditions. The values that I find strange are those of the morning (from 1 am to 9 am). Can anyone explain to me why it deviates in this way from the results of ENVI-met.

Thank you in advance.
![Immagine1](https://user-images.githubusercontent.com/103640974/167134968-87a9f3a3-6a5e-48a9-9e7a-022b79fb7034.png)

**Comment by biglimp** [comment]
Solwieg does not simulate air temperature (Ta) and wind speed (ws) so this has to be input into the model whereas ENVI-met and simulate Ta and ws. Thus, the variation comes from that mainly, is my guess. Also, there is room for improvment concerning mean radiant temperature estimation in solweig during night. We have plans to start working on that. I suggest, check Ta, ws and Tmrt to find the reason for the deviations.

**Reply by SimoneBeltramino** [reply]
Thank you so much for all the information. I checked and it seems that the problem.
